packages/members/layer2/index.js

```diff
@@ -1,29 +1,16 @@
 var Members = require('@tryghost/members-layer1');
 
-var members = Members.create();
-
 function show (el) {
     el.style.display = 'block';
 }
 function hide (el) {
     el.style.display = 'none';
 }
 
-function reload() {
-    location.reload();
-}
-
-function setCookie(token) {
-    if (!token) {
-        document.cookie = 'member=null;path=/;samesite;max-age=0';
-    } else {
-        document.cookie = ['member=', token, ';path=/;samesite;'].join('');
-    }
-    return token;
-}
-
 module.exports = {
-    init: function init() {
+    init: function init(options) {
```

Comment:
Are we using these `options` anywhere currently?

Reply:
```suggestion
    init: function init() {
```